fluent-bit/coralogix/README.md

```diff
@@ -8,24 +8,57 @@ The default values we provide can be overriden according to your needs, the defa
 helm show values coralogix-charts-virtual/fluent-bit-coralogix
 ```
 
-## Installation
+## Installation with default/dynamic app_name and sub_system
+Dynamic App_Name and Sub_System can be any Kubernetes accessible environment variable [namespace, container_name, etc...], 
```

Comment:
* accessible environment variable [namespace, container_name, etc..] -> those are not environment variables 
`App_Name` & `Sub_System` - should be marked